sidebars.js

```diff
@@ -62,6 +62,23 @@ const sidebars = {
     },
     'metrics',
     'examples',
+    {
+      type: 'category',
+      label: 'Tutorials',
+      collapsible: true,
+      collapsed: false,
+        items: [
+          {
+            type: 'category',
+            label: 'Minimal Working Examples',
```

Comment:
Is this category necessary? Since it is the only tutorial, we might put it directly in the `tutorial` category maybe changing its name to something like "react simple example"

Reply:
We have 4 minimal working examples: one for React and the rest for React Native (with Expo), iOS, and Android.